### ISSUE TITLE
Keyboard navigation doesn't work with search-enabled="false"

### ISSUE THREAD

**Issue by mihaic-gf** [angular-ui/ui-select#375]
Hello,

Great job with this project, I have only one small problem, if I add search-enabled="false" when I have few options, keyboard navigation doesn't work anymore. Is there any other way to hide search field?

Example:
ui-select ng-model="person.selected" theme="select2" ng-disabled="disabled" search-enabled="false" style="min-width: 300px;"

Best Regards,
Mihai

**Comment by davideanderson** [comment]
+1

**Comment by jonsonjonson** [comment]
+1

**Comment by mareczek** [comment]
+1

**Comment by kleyy** [comment]
+1

**Comment by uschmann** [comment]
+1

**Comment by brentmiller** [comment]
+1

**Comment by seinavco** [comment]
+1

**Comment by akashta** [comment]
+1

**Comment by chloe-pellen** [comment]
+1

**Comment by LFDM** [comment]
Is there any progress on this?

**Comment by chloe-pellen** [comment]
:/ ????

**Comment by progmars** [comment]
+1
(many times).

This is the only issue which made our UI testers angry. Keyboard navigation is a must, especially in enterprise applications where system operators often navigate forms only using keyboard because that's much faster.

As far as I have seen, ui-select is the most popular (68 contributors!) dropdown element for Angular, but this issue is real showstopper. Maybe I'll find some time to fix it myself, but more likely, I'll find some other dropdown, maybe less popular (and less supported) but with keyboard navigation. Which is a shame, because otherwise ui-select is great.

**Comment by fcaballero** [comment]
@wesleycho there are already some PR that can solve this issue; [PR1114](https://github.com/angular-ui/ui-select/pull/1114) and [PR1109](https://github.com/angular-ui/ui-select/pull/1109). I had the same idea and I think is the most cleanest way as the searchInput already has the keyboard navigation handler.
@cbryer and @cdjackson may you let us know whether those PR are ready to merge?

**Comment by user378230** [comment]
#1114 looks better, @fcaballero if you feel strongly you could fork cbryer's repo, rebase, resolve any conflicts, squash and create a new PR?

**Comment by fcaballero** [comment]
@user378230 ok for me, I will take care of it ;)

**Comment by fcaballero** [comment]
@user378230 or @wesleycho may you please merge these changes in #1543 and remove the oldest PRs?
